tasks/train-a/header-fragment.md

```diff
@@ -0,0 +1,79 @@
+# Site header
+
+## Description
+
+The header serves as the primary navigation and information bar for users. It must be designed to be responsive and user-friendly, providing essential functions and access points to various parts of the application.
+
+### Requirements
+
+#### Requirement 1: Header states
+
+- The header must support three different states based on the user's authentication and authorization level: Guest, General User, and Manager/Administrator.
+
+#### Requirement 2: Guest state
+
+- The header must display the following links for guests:
```

Comment:
a bit confusing ':' at the end of the line :)
maybe to add smth like `you can find requirements in the #### Acceptance Criteria 1: Guest state`